src/Details/ShowDetails.js

```diff
@@ -1,22 +1,42 @@
-import React from 'react';
-import movieData from './../movieData'
+import React, { Component } from 'react';
 import './ShowDetails.css'
+import { fetchSingleMovie } from '../utils/APICalls'
 
-const ShowDetails = ({ movie, handleClick }) => {
-  return (
-    <div className='details-card'>
-    <button className='home-button' onClick={() => handleClick(false)}>Home Page</button>
-      <img className='details-image' src={movie.poster_path}/>
-      <h2 className='details-title'>{movie.title}</h2>
-      <h4 className='movie-release'>{movie.release_date}</h4>
-      <h4 className='movie-rating'>{movie.average_rating}</h4>
-      <h4 className='details-overview'>{movie.overview}</h4>
-      <h4 className='details-genre'>{movie.genres}</h4>
-      <h4 className='budget'>{movie.budget}</h4>
-      <h4 className='revenue'>{movie.revenue}</h4>
-      <h4 className='runtime'>{movie.runtime}</h4>
-      <h4 className='tagline'>{movie.tagline}</h4>
-    </div>
-)}
+class ShowDetails extends Component {
+  constructor({ movie, handleClick }) {
+    super();
+      this.state = {
+        movieID: movie.id,
+        specificMovie: {},
+        movieSelected: movie.id || false,
+        error: ''
+      }
+  }
+
+  render() {
+    return (
+      <div className='details-card'>
+      <button onClick={() => this.handleClick(false)}>Home Page</button>
```

Comment:
How are we passing handleClick through here?  Could we adjust the pathway to regain functionality?

Reply:
Agreed - the naming of this function could be more semantic, as well